.github/workflows/release.yaml

```diff
@@ -0,0 +1,41 @@
+name: Build and relase image
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '1.14.8' # The Go version to download (if necessary) and use.
```

Comment:
Why exactly this pinned version?

Reply:
`go.mod`  is 1.13; [shouldn't it fit ?](https://github.com/AbsaOSS/terraform-controller/blob/master/go.mod#L3).

Reply:
> Why exactly this pinned version?

The one i have installed locally

>go.mod is 1.13; shouldn't it fit ?. otherwise LGTM.

IIUC it doesn't matter. What matters is what you have in go.mod

Reply:
@k0da, ok, can we change to 1.15.0 or makes latest sense? I think we can promote terraform-controller to go1.15  if all works well.

Reply:
sure, but separately please

Reply:
@k0da i fail to follow logic here. If it doesn't matter why did you specify it ?

Reply:
To avoid situations when we pull something what is not tested yet.